src/TouristInfoCards.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Cards from './Cards';
+import faker from 'faker';
+const TouristInfoCards = () => {
+    return (
+        <div className="card-deck">
+            <Cards website="https://peoplemakeglasgow.com/" city="Glasgow" image={faker.image.city()} />
```

Comment:
Huh, that's cool - I didn't realise Faker could provide images too

Reply:
Thanks @40thieves  for you valuable feedback I have update my PR +1